src/components/CreateAccount/CreateAccount.jsx

```diff
@@ -13,19 +13,20 @@ import {
   Heading,
   Stack,
   Box,
-  RadioGroup,
-  Radio,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+  ModalFooter,
+  ModalCloseButton,
+  useDisclosure,
+  Select,
 } from '@chakra-ui/react';
-import { withCookies, Cookies, cookieKeys } from '../../utils/CookieUtils';
+import { SmallAddIcon } from '@chakra-ui/icons';
 import { registerWithEmailAndPassword } from '../../utils/AuthUtils';
 import styles from './CreateAccount.module.css';
-import AUTH_ROLES from '../../utils/AuthConfig';
 import { passwordRequirementsRegex } from '../../utils/utils';
 
-const { SUPERADMIN_ROLE, ADMIN_ROLE, DRIVER_ROLE } = AUTH_ROLES.AUTH_ROLES;
-
-const CreateAccount = ({ cookies }) => {
-  const [role, setRole] = useState(ADMIN_ROLE);
+const CreateAccount = ({ isSuperAdmin, memberType, refreshData }) => {
```

Comment:
We won't be needing `memberType` anymore as we're keeping everything "Staff" to make it easier (I might have repeated this somewhere else in the review).